src/containers/Profile/ProfileDetails.js

```diff
@@ -121,7 +197,8 @@ const mapStateToProps = state => {
     userStats,
     switchValue: profileUI.switchValue,
     currentTab: profileUI.currentTab,
-    loadedUserInfo
+    loadedUserInfo,
+    onEditProfile: props.onEditProfile
```

Comment:
If it is already in the props, what is the purpose of having it here? No state is needed?